subsys/bluetooth/host/Kconfig

```diff
@@ -641,6 +640,21 @@ config BT_CONN_PARAM_UPDATE_TIMEOUT
 	  "The Peripheral device should not perform a Connection Parameter
 	  Update procedure within 5 seconds after establishing a connection."
 
+config BT_CONN_PARAM_RETRY_COUNT
+	int "Peripheral connection parameter update retry attempts"
+	default 3
+	help
+	  This value corresponds to number of times to retry connection
+	  parameter update to attain the preferred value set in GATT
+	  characteristics in the Peripheral.
+
+config BT_CONN_PARAM_RETRY_TIMEOUT
+	int "Peripheral connection parameter update retry timeout in milliseconds"
+	default 5000
+	help
+	  The value is a timeout used by peripheral device to wait until retry
+	  to attempt requesting again the preferred connection parameters.
```

Comment:
Should we set some ranges for these? 

AFAIK these are proper signed integers, and I guess we at the very least should restrict it from being a negative value. 

I also see that `BT_CONN_PARAM_RETRY_COUNT` is stored in `conn_param_retry_countdown`, which is a uint16_t, so the maximum value should also by 65535.